src/OpenLoco/src/Vehicles/Vehicle2.cpp

```diff
@@ -154,10 +155,11 @@ namespace OpenLoco::Vehicles
 
         Vehicle train(head);
         train.cars.applyToComponents([](auto& component) {
-            if (component.var_5E != 0)
+            // If the vehicle is wheel slipping, then its wheel slipping value is incremented.
+            if (component.wheelSlipping != 0)
             {
-                component.var_5E++;
-                component.var_5E &= 0x3F;
+                component.wheelSlipping++;
+                component.wheelSlipping &= 0x3F;
```

Comment:
3F represents the max duration of the wheel slip in ticks. We could introduce a konstant for it